nncf/experimental/torch/sparsify_activations/ActivationSparsity.md

```diff
@@ -0,0 +1,145 @@
+### Activation Sparsity
+
+The `sparsify_activations` algorithm is a post-training method designed to introduce sparsity into the activations of a neural network. This process reduces the number of active neurons during inference by masking out neurons based on their magnitude relative to a calibrated static threshold.
+
```

Comment:
Could you please add a note that this is an experimental algorithm?

Reply:
Added to the title, following https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/KnowledgeDistillation.md